MinecraftClient/ChatBots/AutoAttack.cs

```diff
@@ -40,14 +55,53 @@ public override void Update()
                 attackCooldownCounter = attackCooldown;
                 if (entitiesToAttack.Count > 0)
                 {
-                    SendAnimation(Inventory.Hand.MainHand); // Arm animation
-                    foreach (KeyValuePair<int, Entity> entity in entitiesToAttack)
+                    if (!singleMode)
```

Comment:
Since you have an `else` statement below, you can avoid a double negation by putting the existing code under `if (singleMode)` and the new code under `else` (Currently, the `else` code happens if `not` `not` `singleMode`) 😉